docs/os/windows/windows-vm41.md

```diff
@@ -0,0 +1,203 @@
+Installing a Windows VM
+=======================
+
+Simple Windows install
```

Comment:
IMHO there shouldn't be a specific "Simple Windows install" and then another main "Windows VM installation" section. Maybe make three standalone sections like so ?
- Installation with the GUI
- Installation with `qvm-create-windows-qube`
- CLI installation